src/messages/manifestjson.js

```diff
@@ -165,3 +165,11 @@ export const NO_DEFAULT_LOCALE = {
     See: https://mzl.la/2hjcaEE`),
   file: MANIFEST_JSON,
 };
+
+export const WRONG_ICON_EXTENSION = {
+  code: 'WRONG_ICON_EXTENSION',
+  legacyCode: null,
+  message: _('Consider Adding another icon'),
+  description: sprintf('Icons Should be of type jpg, jpeg, webp, giff, png or svg.'),
```

Comment:
This should not use `sprintf` since you're not using format-strings. You can just use `_('')` to mark the string as translatable.

Also, I'd write something like `Icons should be one of JPG, WebP, GIFF, PNG or SVG` which is what we want eventually.